### PR TITLE
Add a note on now() in backgrounded tabs

### LINKED ISSUE #65: can performance.now() timers be frozen for background tabs, etc?
Currently users in chrome are seeing relatively large divergence from Date.now() and timeOrigin+performance.now().  We believe this is likely due to chrome allowing the performance.now() to counters to be frozen for some periods, like when a tab is in the background.  See:

https://bugs.chromium.org/p/chromium/issues/detail?id=948384

While the spec mentions clock skew and system adjustments, its not clear to me if the browser is allowed to freeze counters for long periods.  This can lead to two tabs for the same site having drastically different times that can no longer be compared.

If we cannot compare across tabs with performance.now(), must sites use Date.now() if they want this kind of comparison?  This would seem unfortunate to me.

### PR DESCRIPTION
**Pull request by yoavweiss**
Closes #65

***
<a href="https://pr-preview.s3.amazonaws.com/yoavweiss/hr-time/pull/69.html" title="Last updated on May 14, 2019, 9:36 PM UTC (02bcea5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/hr-time/69/c0bcbc2...yoavweiss:02bcea5.html" title="Last updated on May 14, 2019, 9:36 PM UTC (02bcea5)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -341,6 +341,10 @@ <h3>Monotonic Clock</h3>
     properties only apply for contexts that can reach other by exchanging
     messages via one of the provided messaging mechanisms - e.g. `postMessage`,
     `BroadcastChannel`, etc.</p>
+    <p class="note">The user agent needs to make sure that the monotonic clock of
+    backgrounded tabs continues to advance at the same speed as the system
```
Comment (npm1) [inline index.html]: I don't think 'same speed as the system clock' is valid. In fact, the monotonic clock is not supposed to be subject to system clock skew.

Comment (igrigorik) [reply index.html]: Some user agents can chose to throttle timers and callbacks in certain conditions, for example when a tab is backgrounded, this should not affect the resolution or accuracy of time passed surfaced by the monotonic clock.

Warmer?

Comment (yoavweiss) [reply index.html]: yup

File: index.html [hunk 2]
```diff
@@ -341,6 +341,10 @@ <h3>Monotonic Clock</h3>
     properties only apply for contexts that can reach other by exchanging
     messages via one of the provided messaging mechanisms - e.g. `postMessage`,
     `BroadcastChannel`, etc.</p>
+    <p class="note">In certain scenarios (e.g. when a tab is backgrounded), the
+    user agent may choose to throttle timers and periodic callbacks run in that
+    context or even freeze them entirely. Any such throttling should not effect
```
Comment (npm1) [inline index.html]:
```suggestion
    context or even freeze them entirely. Any such throttling should not affect
```

**Pull request merged**